src/modules/accommodations/dto/request/create-accommodation.dto.ts

```diff
@@ -29,12 +29,18 @@ export class CreateAccommodationDto {
   @ApiProperty({ default: 'Malibu, California', maxLength: 200 })
   location: string;
 
-  @TransformToNumber()
+  @TransformToDouble()
   @IsNumber()
   @IsNotEmpty()
   @ApiProperty({ default: 200.0, description: 'Price per night' })
   pricePerNight: number;
 
+  @TransformToNumber()
```

Comment:
can you change this name back to TransformToInt. it was like that initially when I wrote it but I made a mistake and changed its name.

Reply:
done